src/sentry/models/commitfilechange.py

```diff
@@ -4,6 +4,8 @@
 
 from sentry.db.models import (BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr)
 
+COMMIT_FILE_CHANGE_TYPES = frozenset(('A', 'D', 'M'))
```

Comment:
Should I also include lower case or should that be illegal?

Reply:
I would start accepting less data. It is easier to add new valid states than remove existing ones.